dcicutils/ff_utils.py

```diff
@@ -1022,7 +1022,7 @@ def get_schemas(key=None, ff_env: Optional[str] = None, *, allow_abstract: bool
     base_url = 'profiles/'
     add_on = 'frame=raw'
     if portal_vapp:
-        full_url = f"{base_url}?{add_on}"
+        full_url = f"/{base_url}?{add_on}"
```

Comment:
i guess this is most conservative. if `/` is ok in get_metadata, then maybe put the `/` in the initialization of `base_url`. But either is OK and this is safest, I guess.